03.asynchronous/query.js

```diff
@@ -0,0 +1,7 @@
+export const createQuery =
+  "CREATE TABLE books (id INTEGER PRIMARY KEY AUTOINCREMENT, title TEXT UNIQUE)";
+export const insertQuery = "INSERT INTO books (title) values ($1)";
```

Comment:
コメントし忘れていましたが、SQL のキーワード部分は大文字で書いてください。

```suggestion
export const insertQuery = "INSERT INTO books (title) VALUES ($1)";
```

Reply:
こちら見落としておりました💦ご指摘ありがとうございます！
以下で対応いたしました。

[7ad4283](https://github.com/yokomaru/js-practices/pull/3/commits/7ad4283d62051a6a1059f9d5551f67160bc95a91)